modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -1,15 +1,18 @@
 = Manage Throughput
-:description: Manage the throughput of Kafka traffic with configurable properties.
+:description: Learn how to manage the throughput of Kafka traffic.
 :page-categories: Management, Networking
 // tag::single-source[]
 
-Redpanda supports throughput throttling on both ingress and egress independently, and allows configuration at the broker and client levels. This helps prevent clients from causing unbounded network and disk usage on brokers.
-
-* Broker-wide limits apply to all clients connected to the broker and restrict total traffic on the broker. 
-* Client limits apply to a set of clients defined by their `client_id` and help prevent a set of clients from starving other clients using the same broker. You can manage client quotas with xref:reference:rpk/rpk-cluster/rpk-cluster-quotas.adoc[`rpk cluster quotas`] or with the Kafka API. When no quotas apply, the client has unlimited throughput. 
+Redpanda supports throughput throttling on both ingress and egress independently, and allows configuration at the broker and client levels. This helps prevent clients from causing unbounded network and disk usage on brokers. You can configure limits at two levels:
 
+* *Broker limits*: These apply to all clients connected to the broker and restrict total traffic on the broker. 
+ifndef::env-cloud[]
+* *Client limits*: These apply to a set of clients defined by their `client_id` and help prevent a set of clients from starving other clients using the same broker. You can manage client quotas with xref:reference:rpk/rpk-cluster/rpk-cluster-quotas.adoc[`rpk cluster quotas`], with {ui}, or with the Kafka API. When no quotas apply, the client has unlimited throughput. 
+endif::[]
 ifdef::env-cloud[]
-NOTE: Throughput throttling is supported for BYOC and Dedicated clusters.
+* *Client limits*: These apply to a set of clients defined by their `client_id` and help prevent a set of clients from starving other clients using the same broker. You can manage client quotas with xref:reference:rpk/rpk-cluster/rpk-cluster-quotas.adoc[`rpk cluster quotas`], with the {ui} UI, with the link:/api/doc/cloud-dataplane/operation/operation-quotaservice_listquotas[Redpanda Cloud Data Plane API], or with the Kafka API. When no quotas apply, the client has unlimited throughput. 
```

Comment:
@JakeSCahill can you please confirm that this is the right way to link into the Cloud API with Bump: `link:/api/doc/cloud-dataplane/operation/operation-quotaservice_listquotas[Redpanda Cloud Data Plane API]`

That's how it's set up in other pages, but it's not generating the link in the preview.